README.md

```diff
@@ -60,8 +60,18 @@ To execute the template:
 (eco-template:index "My Blog" nil)
 ```
 
-**Note:** Eco is designed to be output-agnostic, so by default it will **not**
-autoescape HTML. Use the `e` function for that. You have been warned.
+Eco is designed to be output-agnostic, however, by default it will
```

Comment:
Yes, this is the correct default.

Reply:
By the way, I think this behaviour is still a bit surprising under certain conditions.

Users, I'd guess, will expect explicit (e "string") to escape their strings regardless of the values of ESCAPE-HTML, while "string" (without an explicit escape with eco-template:e) will follow the behaviour specified by ESCAPE-HTML.

Current, the behaviour of ESCAPE-HTML overrides everything, so for example, specifying :ESCAPE-HTML NIL will disable all explicit (e "string") in users' templates.

Reply:
Right -- maybe we should rename `e` to something that better indicates what's actually going on. Like `embed` or some such.